File: day 98/classwork/classwork1.js
function manualForEach(arr, func) {
    for (let i = 0; i < arr.length; i++) {
      func(arr[i]);
    }
  }
  let numbers = [1, 2, 3, 4, 5];
  function printNumber(num) {
    console.log(num);
  }
  manualForEach(numbers, printNumber);

File: day 0030/klasswork/classwork1.py
def manual_count(lst, num):
    count = 0
    for n in lst:
        if n == num:
            count += 1 
    return count

numbers = [1, 2, 3, 2, 5, 2, 6, 2]
number_to_find = 2
result = manual_count(numbers, number_to_find)
print(f"რიცხვი {number_to_find} გამოჩნდა სიაში {result} ჯერ.")
